lib/resources/r4/organization.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: Organization
+field_name_base_url: http://hl7.org/fhir/r4/organization-definitions.html#Organization
+fields:
+  - name: identifier
+    required: 'No'
+    cardinality: 0..*
+    type: Identifier
+    description: The identifier for the Organization.
+    children:
+
+      - name: type
+        required: 'No'
+        cardinality: 0..1
+        type: CodeableConcept
+        description: The type for the identifier that can be used to determine which identifier to use for a specific purpose.
+        url: http://hl7.org/fhir/r4/datatypes-definitions.html#Identifier.type
+        binding:
+          description: The type used to determine which identifier to use for a specific purpose.
+          terminology:
+            - display: Identifier Type Codes
+              system: http://hl7.org/fhir/identifier-type
```

Comment:
I dont see this system.... Did we map it wrong again?

Reply:
removed here https://github.com/cerner/fhir.cerner.com/pull/325/commits/cb6f3d36fa7afcdabc3306d00cfa78017b470b11#diff-b08823bfbc91ea717475d1d2372baf64L22 as it was copy paste error from patient resource.